.changeset/new-owls-refuse.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/core': patch
+'@twilio-paste/theme': patch
+---
+
+[Theme] theme breakpoint values are now represented as `px` values instead of `rem` values
```

Comment:
suggestion: include the tip about tests needing to be updated because of this change if it's relevant to consumers